guides/_subsections/automated-analysis.md

```diff
@@ -21,30 +21,30 @@ for use in the *Dashboard*. Read the section on the
   <li>
     {% include_relative _subsections/common/select-a-recording.md
       select-a-recording-caption="
-        If you do not have any recordings present in the Active Recordings
+        If you do not have any `Recordings` present in the *Active Recordings*
```

Comment:
"Recordings" and "Active Recordings" in code blocks

Reply:
The <i>Active Recordings</i> referred here is for the view window of UI
![image](https://github.com/cryostatio/cryostatio.github.io/assets/56743004/9216ced8-eedc-4f17-af15-bedb9873a075)